frontend/webadmin/modules/uicompat/src/main/resources/org/ovirt/engine/ui/uicompat/LocalizedEnums.properties

```diff
@@ -144,7 +144,7 @@ ActionType___AttachVmToUser=Attach User to VM
 ActionType___ChangeDisk=Change CD
 ActionType___CommitNetworkChanges=Commit Network changes
 ActionType___ConnectStorageToVds=Connect Storage Domain to Host
-ActionType___CreateAllSnapshotsFromVm=Create Snapshot
+ActionType___CreateSnapshotForVm=Create Snapshot
```

Comment:
AFAIU I don't need to rename in the language specific files?

Reply:
No, it will be updated on the next push from Zanata, meanwhile the English translation will be used.